python/oneflow/framework/docstr/utils.py

```diff
@@ -13,13 +13,9 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-_function_docstr = {}
 
-
-def add_docstr(fun, docstr: str):
-    _function_docstr[fun] = docstr
+import oneflow._oneflow_internal
 
 
-def register_docstr():
-    for (fun, docstr) in _function_docstr.items():
-        setattr(fun, "__doc__", docstr)
+def add_docstr(fun, docstr: str):
+    oneflow._oneflow_internal.add_doc(fun, docstr)
```

Comment:
```suggestion
    return oneflow._oneflow_internal.add_doc(fun, docstr)
```

这个地方是不是改成 return 比较好，虽然目前用不上返回值，但是没有丢掉 ` oneflow._oneflow_internal.add_doc` 的功能